pages/workflow/WorkflowHandler.php

```diff
@@ -156,9 +160,9 @@ public function setupIndex($request)
 
         $components = $templateMgr->getState('components');
         $components[FORM_AUDIENCE] = $audienceForm->getConfig();
-        $components[FORM_CATALOG_ENTRY] = $catalogEntryForm->getConfig();
+        $components[FORM_CATALOG_ENTRY] = $this->getLocalizedForm($catalogEntryForm, $latestPublication, $submissionContext);
```

Comment:
Haven't parameters for getLocalizedForm() changed to `form, submissionLocale, locales` ?